terraform/modules/jvs-services/cert-rotator.tf

```diff
@@ -26,13 +26,17 @@ module "cert_rotator_cloud_run" {
 
   region = var.region
   name   = "jvs-cert-rotator"
-  image  = var.jvs_cert_rotator_service_image
+  image  = var.jvs_container_image
+  args   = ["rotation", "server"]
```

Comment:
Since everything is the same binary, we rely on args to spawn the correct process.